File: app/src/main/java/com/gmail/mostafa/ma/saleh/yawa/models/City.java
package com.gmail.mostafa.ma.saleh.yawa.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by Mostafa Saleh on 05/01/2017.
 */

public class City implements Serializable {

    @SerializedName("id")
    @Expose
    public String id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("coord")
    @Expose
    public LatLng coordinates;
    @SerializedName("country")
    @Expose
    public String country;

}

File: app/src/main/java/com/gmail/mostafa/ma/saleh/yawa/models/City.kt
package com.gmail.mostafa.ma.saleh.yawa.models

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by Mostafa Saleh on 05/01/2017.
 */

@Parcelize
data class City(@SerializedName("id")
           @Expose
           val id: String,
           @SerializedName("name")
           @Expose
           val name: String) : Parcelable
